tests/foreman/ui/test_discovery.py

```diff
@@ -275,14 +333,90 @@ def test_update_discovery_rule_2(self):
 
         """
 
-    def test_update_discovery_rule_3(self):
-        """@Test: Update the discovered host name with blank or long name
+    def test_update_discovery_prefix(self):
+        """@Test: Update the discovery_prefix parameter other than mac
 
         @Feature: Foreman Discovery
 
+        @Steps:
+        1. Goto settings -> Discovered tab -> discovery_prefix
```

Comment:
FYI, you can use unicode characters like the [rightwards arrow](http://www.fileformat.info/info/unicode/char/2192/index.htm) so long as the right comment is added to the first line of this file. You're not obliged to, of course, but they do look cool. :)